### PR TITLE
New Contracts Hierarchy

### PR DESCRIPTION
**Pull request by rcinfodesign**

### CODE REVIEW

File: _docs/developers/smartcontracts/documenting.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# Documenting Changes for Deployment
+Smart contract developers who have developed enhancements to the FIO protocol should consider several important deliverables: 
+
+- The developer must at a minimum document the commands necessary to be executed on the previous version of the chain in order to deliver their changes
+- The developer must summarize the necessary dependancies of any modified contracts:
+    - Explain what the order of delivery of the changes must be
+    - Communicate any changes that can be done in any iterations of deployment that might result during test net and main net delivery. 
+- The developer should also clearly communicate changes that MUST go into the chain together.
+
+This documentation should <go where?>
```
Comment (ericbutz) [inline _docs/developers/smartcontracts/documenting.md:17]: If you want to specify location, design docs are kept at: https://fioprotocol.atlassian.net/wiki/spaces/FD/pages/7471298/Development+Documents+Active

**Review by ericbutz — CHANGES_REQUESTED** [review]

File: _docs/developers/smartcontracts/deploying.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+---
+title: Deploying FIO Contracts
+description: Deploying FIO contracts 
+redirect_from:
+   - /docs/developers/smartcontracts
+
+---
+
+# Deploying FIO Contracts
+This document describes the important commands that must be included in a deployment bundle when a FIO smart contract developer is preparing to deliver changes for release to test net and main net. The document also attempts to provide examples of the commands that are frequently used in the deployment bundle for projects delivering new contracts to the FIO protocol.
```
[the remark this answers is not in the document]
Comment (rcinfodesign) [reply _docs/developers/smartcontracts/deploying.md:10]: OK changed

File: _docs/developers/smartcontracts/index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# Overview of FIO Contracts Development
+ The FIO protocol is a smart contract platform that relies on a set of well-defined smart contracts that implement the various functions of the FIO protocol. Maintaining and extending these contracts is the job of smart contract blockchain developers who serve the protocol by completing efforts defined in worker proposals approved by the FIO steering committee.
+ 
+ When new contracts are made by developers, it is important to understand the process of adding a new contract into the set of system contracts that make up the FIO protocol. It is also useful to understand the guidelines and working instructions that help a developer to document any new contracts, or existing contract modifications they might be making.
+
+
+|Content|Summary|
+|---|---|
+|[Guidelines for Documenting Changes]({{site.baseurl}}/docs/developers/smartcontracts/documenting) |Describes documentation deliverables required of developers for any changes and additions to smart contracts|
```
[the remark this answers is not in the document]
Comment (rcinfodesign) [reply _docs/developers/smartcontracts/index.md:16]: Done

File: _docs/developers/smartcontracts/deploying.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+## Set Up Development Environment
+In order to begin the process of rollout testing, the developer must first set up a development environment in a configuration that matches the present main net release. The developer must use a branch of fio.devtools which they have used for development, but this branch must be further modified for rollout testing.
```
[the remark this answers is not in the document]
Comment (rcinfodesign) [reply _docs/developers/smartcontracts/deploying.md:13]: OK I changed; Note that there was reference to updating your "rollout guide" for rollout of your project to the mainnet... I changed the rollout guide to release guide, but kept the "rollout of your project to the mainnet."  LMK if that works!

[the remark this answers is not in the document]
Comment (rcinfodesign) [reply _docs/developers/smartcontracts/deploying.md:13]: nvm-- the next comment references!

File: _docs/developers/smartcontracts/deploying.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+- Next rename the fio.contracts.mainnet directory to be fio.contracts.
+- Now, start up your local dev environment.
+- Run any smoke tests necessary to verify that the local dev environment is up and running.
+
+## Sample Commands
+Once the changes have been tested, the developer can discover, or perform the set of commands used to deploy their changes onto the main net blockchain.
+
+**Note:** The following sequence of commands is for example purposes. The commands necessary are different for any project being delivered and these commands must be discovered and verified by the smart contract developer who develops the changes for the project to be delivered.
+
+The first set of commands necessary to upgrade the main net chain performs the adding of new contract actions, and adds their associated fees to the main net chain. Please be sure to list out in your rollout guide the add action and create fee commands necessary for the rollout of your project to the main net FIO protocol.
```
[the remark this answers is not in the document]
Comment (rcinfodesign) [reply _docs/developers/smartcontracts/deploying.md:34]: Changed.

File: _docs/developers/smartcontracts/deploying.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+Create the new fees
+../fio/build/bin/clio -u http://localhost:8879 push action -f fio.fee createfee '{"end_point":"stake_fio_tokens","type":"1","suf_amount":"3000000000"}' --permission fio.fee@active
+
+../fio/build/bin/clio -u http://localhost:8879 push action -f fio.fee createfee '{"end_point":"unstake_fio_tokens","type":"1","suf_amount":"3000000000"}' --permission fio.fee@active
+
+```
+ 
+
+After adding your new actions and fees to the protocol, it is common to next deploy modified contracts to the main net chain. Be sure you understand all of the contracts impacted by your changes, and the order in which they must be deployed onto main net. Specify the set of commands to be executed to deliver all contract modifications.
```
[the remark this answers is not in the document]
Comment (rcinfodesign) [reply _docs/developers/smartcontracts/deploying.md:67]: Makes sense

**Review by ericbutz — APPROVED** [review]

**Pull request merged**